Integrations/ARIA/playbook_aria_packet_intelligence_test.yml

```diff
@@ -0,0 +1,3000 @@
+id: 0067dcf7-c06e-4565-8143-8c75eeb2e2f0
```

Comment:
This file should be under Playbooks directory

Reply:
Thanks. For clearance, is it TestPlaybooks or Playbooks?

Reply:
TestPlaybooks